.goreleaser.yml

```diff
@@ -132,3 +136,66 @@ dockers:
       - "--label=org.opencontainers.image.version={{.Version}}"
       - "--label=org.opencontainers.image.url=https://github.com/vmware/govmomi/vcsim"
       - "--platform=linux/amd64"
+  - image_templates:
+      - "vmware/govc:{{ .Tag }}-arm64v8"
+      - "vmware/govc:v{{ .Major }}-arm64v8"
+      - "vmware/govc:v{{ .Major }}.{{ .Minor }}-arm64v8"
+      - "vmware/govc:latest-arm64v8"
+    dockerfile: Dockerfile.govc
+    goarch: arm64
+    build_flag_templates:
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--label=org.opencontainers.image.url=https://github.com/vmware/govmomi/govc"
+      - "--platform=linux/arm64/v8"
+  - image_templates:
+      - "vmware/vcsim:{{ .Tag }}-arm64v8"
+      - "vmware/vcsim:v{{ .Major }}-arm64v8"
+      - "vmware/vcsim:v{{ .Major }}.{{ .Minor }}-arm64v8"
+      - "vmware/vcsim:latest-arm64v8"
+    dockerfile: Dockerfile.vcsim
+    goarch: arm64
+    build_flag_templates:
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--label=org.opencontainers.image.url=https://github.com/vmware/govmomi/vcsim"
+      - "--platform=linux/arm64/v8"
+docker_manifests:
+  - name_template: vmware/govc:{{ .Tag }}
+    image_templates:
+    - vmware/govc:{{ .Tag }}
```

Comment:
can we please also append `amd64` to (all of) these images for consistency and so it's clear in the manifest we're building for `amd` and `arm64`?